backend/src/main/resources/webui/popup/switchboardpopup.js

```diff
@@ -16,9 +16,26 @@ var Zepto=function(){function L(t){return null==t?String(t):j[S.call(t)]||"objec
 
 var $ = window.$ = Zepto;
 
+var switchboardURL = null;
+
+function setSwitchboardURL(url) {
+    if (url) {
+        switchboardURL = url;
+        return;
+    } else {
+        for (const x of document.getElementsByTagName('script')) {
+            if (x.src && x.src.includes('switchboardpopup')) {
+                const url = x.src;
+                const end = url.indexOf('/popup/');
+                switchboardURL = url.substr(0, end);
+                return;
+            }
+        }
+        console.error("Could not find automatically set the Switchboard URL. Switchboard invocations will fail.");
```

Comment:
A fall back to the production URL could go here. Configurable via environment variable would be great but I think that for now this could just default to an hardcoded string and then:
`console.error("Could not find automatically set the Switchboard URL. Switchboard invocations will default to: ...");`